pkgs/by-name/gz/gz-cmake/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  ruby,
+  doxygen,
+  graphviz,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gz-cmake";
+  version = "4.1.1";
+
+  src = fetchFromGitHub {
+    owner = "gazebosim";
+    repo = "gz-cmake";
+    tag = "gz-cmake${lib.versions.major finalAttrs.version}_${finalAttrs.version}";
```

Comment:
DRY: Please move that logic into an intermediate variable, in a let/in close before calling stdenv.mkDerivation